src/config/apps/nabla.ts

```diff
@@ -15,10 +15,13 @@ export type NablaConfig = AppConfig &
 
 export const nablaConfig: NablaConfig = {
   tenants: [TenantName.Foucoco],
+  environment: ['staging', 'development'],
```

Comment:
This indicates for which environments Nabla should be shown? Or what is it for?

Reply:
Exactly. Can add a comment there.